src/main/java/sirius/tagliatelle/rendering/GlobalRenderContext.java

```diff
@@ -88,6 +119,29 @@ public GlobalRenderContext(Tagliatelle engine) {
     protected void outputRaw(String string) {
         if (string != null) {
             buffer.append(string);
+
+            if (debugLevel != DebugLevel.OFF) {
+                Pattern pattern = Pattern.compile("<script.*?>");
```

Comment:
Defining the pattern as a constant at the top of the class would improve performance by only compiling it one.
This also applies to the patterns below.

For example:
`private static final Pattern OPENING_SCRIPT_TAG = Pattern.compile("<script.*?>")`

Reply:
Good point. Will also handle script & tag elements case insensitive